library/rendering/entity_rendering/src/main/java/org/quiltmc/qsl/rendering/entity/api/client/ArmorRenderingRegistry.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.rendering.entity.api.client;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.client.render.entity.model.BipedEntityModel;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.ItemConvertible;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.Identifier;
+
+import org.quiltmc.qsl.base.api.event.Event;
+import org.quiltmc.qsl.rendering.entity.impl.client.ArmorRenderingRegistryImpl;
+
+/**
+ * Provides methods for modifying the rendering of worn armor on entities.
+ */
+@Environment(EnvType.CLIENT)
+public final class ArmorRenderingRegistry {
+	private ArmorRenderingRegistry() {
+		throw new UnsupportedOperationException("ArmorRenderingRegistry only contains static declarations.");
+	}
+
+	/**
+	 * The identifier of the default phase for texture/model providers.
+	 */
+	public static final Identifier DEFAULT_PHASE = Event.DEFAULT_PHASE;
```

Comment:
This is a leftover from back when this API used phases directly instead of using events.
Should this be kept, though? The events themselves aren't exposed.

Reply:
Hm, maybe not?